packages/website/md/docs/json_schemas/2/schemas.md

```diff
@@ -1,40 +1,40 @@
 Basic Schemas
 
-*   [Address type](https://github.com/0xProject/0x-monorepo/blob/development/packages/json-schemas/schemas/address.json)
-*   [Number type](https://github.com/0xProject/0x-monorepo/blob/development/packages/json-schemas/schemas/number.json)
-*   [Hex type](https://github.com/0xProject/0x-monorepo/blob/development/packages/json-schemas/schemas/hex.json)
-*   [JS number](https://github.com/0xProject/0x-monorepo/blob/development/packages/json-schemas/schemas/js_number.json)
+*   [Address type](https://github.com/0xProject/0x-monorepo/blob/5ec4b27200297708298deca97603849a37b2f66a/packages/json-schemas/schemas/address_schema.json)
```

Comment:
Will these need to be updated over time?

Reply:
Yes, but I'd rather we update it periodically then have a change in Development (e.g schema file rename) break the links on doc reference pages.